src/client/content_mapblock.cpp

```diff
@@ -216,6 +220,8 @@ void MapblockMeshGenerator::drawCuboid(const aabb3f &box,
 		2, 6, 4, 0
 	};
 
+	static const f32 texture_offset = 1.f / 8192.f;
```

Comment:
static const constexpr (to calculate at compilation time)

Reply:
`constexpr` implies `const`